lib/parser/livesplit_core_parser.rb

```diff
@@ -69,12 +69,15 @@ def self.parse(run, fast: false)
       (0...run.attempt_history_len).each do |i|
         attempt = run.attempt_history_index(i)
         attempt_id = attempt.index.to_i
-        time = attempt.time()
+        time = attempt.time
 
         run_object[:history] << {
-          attempt_number: attempt_id,
+          attempt_number:       attempt_id,
+          realtime_duration_ms: time.real_time.try(:total_seconds).try(:*, 1000) || 0,
           gametime_duration_ms: time.game_time.try(:total_seconds).try(:*, 1000) || 0,
-          realtime_duration_ms: time.real_time.try(:total_seconds).try(:*, 1000) || 0
+
+          started_at: attempt.started && DateTime.parse(attempt.started.to_rfc3339),
+          ended_at:   attempt.ended   && DateTime.parse(attempt.ended.to_rfc3339)
```

Comment:
Both .started and .ended unlike the other methods return owned objects that you need to manage yourself. Or at least the GC will have to, but I'm not sure if we ever confirmed the GC integration works. So you probably want to call .dispose on what is being returned from them. (The general rule is that you need to call dispose on anything that doesn't have a Ref or RefMut postfix: https://livesplit.org/livesplit-core/classes/attemptref.html#started )

Reply:
This should also be addressed just in case so we're not leaking memory.